cmd/install.go

```diff
@@ -45,10 +51,19 @@ func installCommandAction(cmd *cobra.Command, _ []string) error {
 			return errors.Wrap(err, "locating package root failed")
 		}
 	}
-
-	m, err := packages.ReadPackageManifestFromPackageRoot(packageRootPath)
-	if err != nil {
-		return errors.Wrapf(err, "reading package manifest failed (path: %s)", packageRootPath)
+	var manifest *packages.PackageManifest
+	if zipPathFile != "" {
```

Comment:
Nit. Consider having some kind of factory that provides an object with the methods for each kind of package to install. Or two different methods depending on the method to use. So we avoid these ifs here.
We can also refactor this in a follow up.

Reply:
Added factory, now this method looks like much simpler, thanks!